### PR TITLE
feat: add script for updating file passport variables

### PR DESCRIPTION
**Pull request by jessicamcinchak**
For Upload & UploadAndLabel component types, we want to replace their passport variables based on a lookup list. In cases of UploadAndLabel, rules _should not_ be impacted, only the passport variable or "data field" for each document slot.

Test: 
- `pnpm start:fileNames`
- Try it locally against a flow like `opensystemslab/prior-approval-files`

### CODE REVIEW

**Comment by jessicamcinchak** [comment]
[Update document types and add secondary schema for "files" when provided as data (eg Heritage Statements)](https://trello.com/c/kOBotqGJ/2755-update-document-types-and-add-secondary-schema-for-files-when-provided-as-data-eg-heritage-statements)

**Comment by DafyddLlyr** [comment]
@jessicamcinchak Something I just realised - there are also a few places in code where we need to also make this update - e.g. mocks in `planx-core` used for building payloads and functions which rely on hardcoded values such as `extractTagsFromPassportKey()`.

Most of these should be a slightly repetitive find and replace I'd guess?

File: client.js [hunk 1]
```diff
@@ -62,8 +62,25 @@ class Client {
...
+        }
+      }`,
+      {
+        flowId: flowId,
+        liveData: liveFlowData,
+      }
+    );
+    if (errors || !data) throw new Error(formatJSON({ data, errors }));
+    return { ...data };
+  }
```
Comment (jessicamcinchak) [inline client.js:80]: many `-files` flows used as external portals aren't published and we don't necessarily want to publish them before running this script - so now it handles the update mutation conditionally depending if a flow is published or not!

File: fileNames/helpers.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          newFlowData[uploadAndLabelNodeId]["data"]["fileTypes"][i]["fn"] = filesNowToNext[currentFn];
+        }
+      });
+    });
+  return newFlowData;
+}
+
+/**
+ * Find each TextInput (110) or DateInput (120) node in a flow (live or published) that asks about what a file contains and update its' passport variable (fn)
+ */
```
Comment (jessicamcinchak) [inline fileNames/helpers.js:37]: See this flow as example of these "supporting inputs" for files: https://editor.planx.dev/opensystemslab/certificate-of-lawfulness-documents-immunity

**Review by DafyddLlyr — APPROVED** [review]
Working as expected locally! ✅ 

Nice to see this repo grow and prove more and more useful!

**Pull request merged**